pywt/_extensions/_pywt.pyx

```diff
@@ -574,6 +607,255 @@ cdef public class Wavelet [type WaveletType, object WaveletObject]:
                            filter_bank=self.filter_bank)
 
 
+cdef public class ContinuousWavelet [type ContinuousWaveletType, object ContinuousWaveletObject]:
+    """
+    ContinuousWavelet(name) object describe properties of
+    a continuous wavelet identified by name.
+
+    In order to use a built-in wavelet the parameter name must be
+    a valid name from the wavelist() list.
+
```

Comment:
this is the main docstring where users will start, so would be good to add ``Attributes`` and ``Methods`` sections (in format like [here](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt)), and note that the ``wavefun`` method has usage info/examples.